parser.py

```diff
@@ -0,0 +1,45 @@
+# Author: Pankil Shah
+# Created on: 03/13/2021
+# Description: Parse a corpus and dump the sentences into the database 
+
+from nltk.corpus import brown
+import nltk.data
+import logic
+import datetime
+from os import listdir
+from os.path import isfile, join
+
+#print(brown.tagged_sents()[0:2])
+
+#dirPath = 'E:\\SLU\\Sem1\\Principles of SD\\Project Material\\brown'
+path = input("Enter path of the corpus directory: ")
+language = input("Enter language: ")
+#language = "english"
+
+# list of all file names in the corpus directory
+fileNameList = [f for f in listdir(path) if isfile(join(path, f))]
+languageId = str(logic.SQLQuery("select Lang_ID from lang_ref where Lang_Desc = '" + language + "'")[0])
+
+for fileName in fileNameList:
+
+    # ignore the files CONTENTS and README
+    if (fileName != "CONTENTS") and (fileName != "README"):
```

Comment:
This loops formatting looks good, this will speed up the program by ignoring the unnecessary files.